.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,4 @@
+contact_links:
```

Comment:
Could we please include blank issues as well?

```suggestion
blank_issues_enabled: true
contact_links:
```

Reply:
I think this is set to `true` by default. The blank issue option shows up in my repository -

![image](https://user-images.githubusercontent.com/74055102/179909486-c322bef9-248d-4ebe-9fc1-c37d6207ae1a.png)

Updating it, just in case GitHub decides to turn the default value to `false`.

Reply:
Thanks Saransh! 🙏

Yeah explicit is better than implicit 🙂